packages/metadata/src/metadata-service.ts

```diff
@@ -249,7 +249,7 @@ export class MetadataService implements IMetadataService {
     /**
      * Returns all metadata root elements.
      *
-     * @returns
+     * @returns map of metadata element
```

Comment:
Add more details about what kind of mapping it is.

Reply:
updated